src/commands/entities/configure.ts

```diff
@@ -167,21 +167,17 @@ the corresponding property in the entity is not modified.
 
     const {
       'entity-type': entityType,
-      'has-a': hasA,
-      'is-a': isA,
       locale,
       pattern,
     } = options
 
-    // Entity types 'regex' and 'relational' require mandatory flags.
+    // Entity types 'relational' require mandatory flags upon creation or
```

Comment:
Is there supposed to be more than one type here? The grammar is still plural, but there's just the one relational type.

Reply:
Corrected.